setup.py

```diff
@@ -1,17 +1,24 @@
-from os import chdir
-from os.path import abspath, dirname, join, normpath
+import os
 from setuptools import find_packages, setup
 
 import service_objects
 
 
-def read_file(filename):
-    with open(join(dirname(abspath(__file__)), filename)) as f:
-        return f.read()
+BASE_DIR = os.path.dirname(__file__)
+README_PATH = os.path.join(BASE_DIR, 'README.md')
 
+try:
+    import pypandoc
+    LONG_DESCRIPTION = pypandoc.convert(README_PATH, 'rst')
```

Comment:
If you do convert the mardown into RST, you should update long_description_content_type to "text/x-rst; charset=UTF-8"